### PR TITLE
irrlichtmt: update to 1.9.0mt8

### PR DESCRIPTION
**Pull request by aeiouaeiouaeiouaeiouaeiouaeiou**
* remove platforms line

###### Type(s)

- [ ] bugfix
- [ ] enhancement
- [ ] security fix

###### Tested on

macOS 10.8.5 12F2560 x86_64
Xcode 5.1.1 5B1008

###### Verification 
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [ ] referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL? 
- [x] checked your Portfile with `port lint --nitpick`?
- [ ] tried existing tests with `sudo port test`?
- [x] tried a full install with `sudo port -vst install`?
- [x] tested basic functionality of all binary files?
- [ ] checked that the Portfile's most important [variants](https://trac.macports.org/wiki/Variants) haven't been broken?

### CODE REVIEW

File: devel/irrlichtmt/Portfile [hunk 1]
```diff
@@ -4,18 +4,17 @@ PortSystem          1.0
 PortGroup           github 1.0
 PortGroup           cmake 1.1
 
-github.setup        minetest irrlicht 1.9.0mt4
+github.setup        minetest irrlicht 1.9.0mt8
 name                irrlichtmt
 categories          devel
-platforms           darwin
```
Comment (l2dy) [inline devel/irrlichtmt/Portfile:10]: Why would you need to remove this?

Comment (herbygillot) [reply devel/irrlichtmt/Portfile:10]: @l2dy `platforms darwin` is now implicit to MacPorts, and is being removed from ports by default.

**Comment by l2dy** [comment]
minetest 5.6.1 could not be built with this version of IrrlichtMt. Is there a bug?

```
In file included from .../work/minetest-5.6.1/src/client/shader.cpp:43:
/opt/local/include/irrlichtmt/mt_opengl.h:11:10: fatal error: 'KHR/khrplatform.h' file not found
#include <KHR/khrplatform.h>
         ^~~~~~~~~~~~~~~~~~~
```

**Comment by l2dy** [comment]
Each version of minetest has a recommended version of IrrlichtMt. If other versions are used, minetest may fail to build, which is happening for the current version (5.5.1) in the ports tree. Please update the minetest port as soon as possible.

**Comment by aeiouaeiouaeiouaeiouaeiouaeiou** [comment]
@l2dy I'll try to push changes today.

**Pull request merged**